### ISSUE TITLE
Warn if `param` in kdoc present several times

### ISSUE THREAD

**Issue by kgevorkyan** [saveourtool/diktat#1074]
## Describe the bug
Consider the following code, where there is exist extra `kdoc` for already described `param2`
```kotlin
/**
 * @param param1 description1
 * @param param2 description2
 * @param param2
 */
fun foo(param1: Long, param2: Int) {
    //
}
```

## Expected behavior
`WARN: param2 param already present in argument list`

## Observed behavior
No warnings

## Environment information
* diktat version: 1.0.0-rc3

**Comment by nulls** [comment]
Also, if `@param` present in kdoc for class, but `@property` is required -- Diktat adds a proper one, but doesn't detect `@param` as an issue
